src/index.html

```diff
@@ -1,12 +1,313 @@
+             class="card__image"
+             alt="imac with color screen">
+        <p class="card__nameProduct">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__codeProduct">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="price-name">Price:</p>
+          <p class="price-value">$2,199</p>
+        </div>
+        <a href="#"
+           class="card__button"
+           data-qa="card-hover">
+          <p>BUY</p>
+        </a>
+      </section>
+      <section class="card"
+               data-qa="card">
+        <img src="./images/imac.jpeg"
+             class="card__image"
+             alt="imac with color screen">
+        <p class="card__nameProduct">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__codeProduct">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="price-name">Price:</p>
+          <p class="price-value">$2,199</p>
+        </div>
+        <a href="#"
+           class="card__button"
+           data-qa="card-hover">
+          <p>BUY</p>
+        </a>
+      </section>
+      <section class="card"
+               data-qa="card">
+        <img src="./images/imac.jpeg"
+             class="card__image"
+             alt="imac with color screen">
+        <p class="card__nameProduct">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__codeProduct">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="price-name">Price:</p>
+          <p class="price-value">$2,199</p>
```

Comment:
due to BEM it should be `card__price-name` and `card__price-value`, change in every place where you use them